libobs/obs-scene.c

```diff
@@ -1475,16 +1476,29 @@ static inline void mix_audio(float *p_out, float *p_in, size_t pos,
 		*(out++) += *(in++);
 }
 
-static bool scene_audio_render(void *data, uint64_t *ts_out,
-			       struct obs_source_audio_mix *audio_output,
-			       uint32_t mixers, size_t channels,
-			       size_t sample_rate)
+static inline struct scene_source_mix *get_source_mix(struct obs_scene *scene,
```

Comment:
This should be able to be replaced with `da_find()` right?

Reply:
No, `da_find` requires you to know the entire struct.